msteams-platform/bots/how-to/conversations/conversation-messages.md

```diff
@@ -13,7 +13,7 @@ Each message in a conversation is an `Activity` object of type `messageType: mes
 Basic conversations are handled through the Bot Framework connector, a single REST API. This API enables your bot to communicate with Teams and other channels. The Bot Builder SDK provides the following features:
 
 * Easy access to the Bot Framework connector.
-* Additional functionality to manage conversation flow and state.
+* Another functionality to manage conversation flow and state.
```

Comment:
Why did we make this change? The new phrasing sounds awkward, maybe omit "Another" entirely and just say "Functionality to manage conversation flow and state"?

Reply:
@v-npaladugu,

Additional is the right word. Please revert. Kindly, gte the PRs reviewed by me before we reach out to SMEs.